Frends.SFTP.ListFiles/Frends.SFTP.ListFiles/Enums/HostKeyAlgorithms.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Frends.SFTP.ListFiles.Definitions;
```

Comment:
Namespace should be Enums, since the files are in Enums-folder, not in Definitions.